ramp-tps/Cargo.toml

```diff
@@ -16,6 +16,7 @@ reqwest = { version = "0.9.22", default-features = false }
 serde = "1.0"
 serde_json = "1.0"
 serde_yaml = "0.8.11"
+solana-core = { git = "https://github.com/solana-labs/solana", tag = "v0.21.0" }
```

Comment:
I mostly want to move Rampy into the main repo at this point.  Otherwise it's gonna be breaking all the time (like it has in the past).   But unblocking TdS is more important so 🙈